scripts/download_openmeteo_2024_2025.py

```
@@ -0,0 +1,149 @@
+            "--source", "openmeteo",
+            "--country", "JP",
+            "--start", start_date,
+            "--end", end_date
+        ]
+        
+        print(f"  Starting download at {datetime.now().strftime('%H:%M:%S')}")
+        print(f"  Command: {' '.join(cmd)}")
+        
+        try:
+            # Run subprocess with real-time output
+            process = subprocess.Popen(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)